docs/source/contributor-guide/communication.md

```diff
@@ -68,6 +68,11 @@ Unfortunately, due to spammers, the ASF Slack workspace requires an invitation
 to join. We are happy to invite you -- please ask for an invitation in the
 Discord server.
 
+### Job Board
+
+There are plenty of opportunities to work on DataFusion advertised on the
```

Comment:
A  pedantic suggestion is that many of the jobs I have seen are building some product *with* DataFusion, rather than working only *on* DataFusion.

```suggestion
There are plenty of opportunities to work with DataFusion advertised on the
```